--- C:/Users/ddk22/miniconda3/Lib/site-packages/mgz/fast/header.py	2026-06-18 11:51:05.072934800 -0700
+++ .replay_scratch/mgz/fast/header.py	2026-06-18 12:52:41.340783200 -0700
@@ -468,6 +468,8 @@
             data.read(8)
         if save >= 64.3:
             data.read(4)
+        if save >= 67.0:
+            data.read(4)
 
         players.append(dict(
             number=number,
@@ -548,6 +550,8 @@
         c = unpack('<I', data)
         data.read(12)
         data.read(c * 4)
+    if save >= 67.0:
+        data.read(8)
     if not skip:
         de_string(data)
         data.read(8)
